examples/get_morphologies.py

```diff
@@ -18,13 +18,25 @@
 import pandas as pd
 import os
 import re
+import ssl
 import json
 from urllib.request import urlopen, Request
 from pathlib import Path
 
 NEUROMORPHO_URL = "http://neuromorpho.org"
 
 
+def get_url(url):
+    """Open a URL without SSL verification."""
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
+    ctx.set_ciphers("DEFAULT@SECLEVEL=1")
+    req = Request(url)
+    response = urlopen(req, context=ctx)  # pylint: disable=consider-using-with
```

Comment:
Why not using a context manager as suggested by pylint?

Reply:
I don't know, I can add that yes

Reply:
it's the same as in morphology-workflow example, I don't care, I'll leave it as is

Reply:
Yeah ok, I don't really know the point of a context manager in this context.